config/currency_non_iso.yml

```diff
@@ -94,3 +94,18 @@ gbx:
   thousands_separator: ","
   iso_numeric: ''
   smallest_denomination: 1
+usdc:
+  priority: 100
+  so_code: USDC
+  name: USD Coin
+  symbol: USDC
+  disambiguate_symbol: USDC
+  alternate_symbols: []
+  subunit: Cent
+  subunit_to_unit: 1000000,
```

Comment:
I think this brings up the discussion of the guidance around storing monetary values in the database. At the moment we are using `DECIMAL(21,3)`.

This will not be enough to represent this without a loss of precision.

cc @csalvato - I think you had brought up the topic before.

Reply:
@robinbrandt a missing currency has `subunit_to_unit: 100` so this would be consistent with the existing behaviour

Reply:
yup that's right the above PR is where I got this information from. 👍🏽